sdk/core/Azure.Core/tests/TestFramework/TestEnvironment.Recorded.cs

```diff
@@ -11,32 +11,41 @@ namespace Azure.Core.Testing
     public partial class TestEnvironment
     {
         private TestRecording _recording;
-        private bool _playback;
+        private RecordedTestMode _mode;
 
-        public void SetRecording(TestRecording recording, bool playback)
+        public void SetMode(RecordedTestMode mode)
+        {
+            _mode = mode;
```

Comment:
Should it be allowed to call `SetMode` multiple times?

Reply:
I don't know why not.